ui/components/app/modals/contract-details-modal/index.scss

```diff
@@ -6,6 +6,8 @@
     border-bottom: 1px solid var(--color-border-muted);
 
     &__contract {
+      align-items: center;
```

Comment:
Should be able to use style utility props instead of styles here
`alignItems={AlignItems.center}`

Reply:
Done.